src/modules/cpu/kernel/pre_emphasis_filter.hpp

```diff
@@ -50,7 +50,7 @@ RppStatus pre_emphasis_filter_host_tensor(Rpp32f *srcPtr,
         dstPtrTemp[0] = srcPtrTemp[0] - coeff * border;
 
         Rpp32s vectorIncrement = 8;
-        Rpp32s alignedLength = (bufferLength / 8) * 8;
+        Rpp32s alignedLength = (bufferLength / 8) * 8 - 8;
```

Comment:
Not sure why aligned length is computed like this? >>3 is good for /8 (pow of 2). why -8?

Reply:
Not in the PR